tests/collectors.py

```diff
@@ -87,8 +87,12 @@ def __init__(self, name, parent, module, inventory, tf_var_files,
     self.extra_files = extra_files
 
   def runtest(self):
-    s = plan_validator(self.module, self.inventory, self.parent.path.parent,
-                       self.tf_var_files, self.extra_files)
+    summary = plan_validator(self.module, self.inventory, self.parent.path.parent,
+                             self.tf_var_files, self.extra_files)
+    print(f'\n{self.inventory}')
+    print(yaml.dump({'values': summary.values}))
+    print(yaml.dump({'counts': summary.counts}))
+    print(yaml.dump({'outputs': summary.outputs}))
```

Comment:
Can we remove these prints? You can use tools/plan_summary.py for the same purpose.

Reply:
Use of that tool is not immediate... can we replace those prints with a single one that spits out the command to use the plan summary tool? It would save me a few brain cycles every time :)

Reply:
I added exact `python tools/plan_summary.py` command to run to regenerate inventory.